src/components/pages/Utbetaling/Utbetaling.jsx

```diff
@@ -21,41 +22,52 @@ const Utbetaling = withBehandlingContext(({ behandling }) => {
     } = behandling.utbetaling;
     return (
         <Panel className="Utbetaling" border>
-            <Undertittel className="panel-tittel">Utbetaling</Undertittel>
+            <Undertittel className="panel-tittel">
+                {utbetalingstekster('tittel')}
+            </Undertittel>
             <ListeItemBolk
-                label="Refusjon til arbeidsgiver"
+                label={utbetalingstekster('refusjon')}
                 value={refusjonTilArbeidsgiver ? 'Ja' : 'Nei'}
             />
             <ListeItemBolk
-                label="Betaler arb.giverperiode"
+                label={utbetalingstekster('betaler')}
                 value={betalerArbeidsgiverperiode ? 'Ja' : 'Nei'}
             />
 
             <ListeSeparator type="dotted" />
 
             <ListeItem
-                label="Sykepengegrunnlag"
+                label={utbetalingstekster('sykepengegrunnlag')}
                 value={toKronerOgØre(sykepengegrunnlag)}
             />
-            <ListeItem label="Dagsats" value={toKronerOgØre(dagsats)} />
+            <ListeItem
+                label={utbetalingstekster('dagsats')}
+                value={toKronerOgØre(dagsats)}
+            />
 
             <ListeSeparator type="dotted" />
 
-            <ListeItem label="Antall dager" value={antallDager} />
-            <ListeItem label="Sykmeldingsgrad" value={`${sykmeldingsgrad}%`} />
+            <ListeItem
+                label={utbetalingstekster('dager')}
+                value={antallDager}
+            />
+            <ListeItem
+                label={utbetalingstekster('sykmeldingsgrad')}
+                value={`${sykmeldingsgrad}%`}
+            />
 
             <ListeSeparator type="dotted" />
 
             <ListeItemBolk
-                label="Utbetaling"
+                label={utbetalingstekster('utbetaling')}
                 value={toKronerOgØre(antallDager * dagsats)}
             />
 
             <ListeSeparator type="solid" />
 
-            <Undertittel>Filter for MVP</Undertittel>
+            <Undertittel>Hva dekkes av løsningen nå</Undertittel>
```

Comment:
👍 for å fjerne "MVP" fra skjermbilder :) Kan kanskje flyttes til tekster.js?

Reply:
Har den i tekster.js her: `bokmål.mvp = 'Hva dekkes av løsningen nå';`
Glemte å bruke den i `Utbetaling`. Bra spotta :)